src/server/game/map/tile/improvement.ts

```diff
@@ -72,13 +107,13 @@ export class Improvement {
       traderCount--;
     }
 
-    const totalYield = this.yield.add(baseYield);
-    this.storage.incr(totalYield);
+    this.storage.incr(this.yield);
     this.storage.cap();
   }
 
   store(resources: Yield): void {
     this.storage.incr(resources);
+    this.errand?.storedThisTurn.incr(resources);
```

Comment:
Wouldn't this duplicate the resources?

Reply:
No, that's only a counter to keep track of how much has been stored this turn. It resets every turn.